src/bootstrap.tsx

```diff
@@ -287,6 +287,17 @@ const setUserToStore = async (user?: User) => {
   store.dispatch(setUser(user));
 };
 
+const setLoadingImage = (path = './shogun_spinner.gif', id: number) => {
```

Comment:
Non blocking, but it's more convenient to place the optional arguments at the end of the signature:

```suggestion
const setLoadingImage = (id: number, path = './shogun_spinner.gif') => {
```